Resources/config/default_settings.yml

```diff
@@ -1,6 +1,8 @@
 parameters:
     ez_platformui.default.yui: {}
     ez_platformui.default.yui.filter: "min"
+    ez_platformui.default.yui.combine: true
+    ez_platformui.default.yui.build_dir: /bundles/ezplatformuiassets/vendors/yui3/build/
```

Comment:
well, the thing is the whole JS module settings system was made siteaccess aware so for consistency I guess that's OK